src/main/java/com/eva/logic/commands/AddApplicationCommand.java

```diff
@@ -53,6 +64,16 @@ public CommandResult execute(Model model) throws CommandException {
 
         Applicant applicantToUpdate = lastShownList.get(index.getZeroBased());
         model.addApplicantApplication(applicantToUpdate, applicationToAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, applicantToUpdate.getName()));
+        if (panelState.equals(APPLICANT_LIST)) {
+            model.updateFilteredApplicantList(PREDICATE_SHOW_ALL_APPLICANTS);
+        } else if (panelState.equals(APPLICANT_PROFILE)) {
+            model.updateFilteredApplicantList(PREDICATE_SHOW_ALL_APPLICANTS);
+            Applicant applicantToView = lastShownList.get(index.getZeroBased());
+            model.setCurrentViewApplicant(new CurrentViewApplicant(applicantToView));
+        }
+        model.setPanelState(APPLICANT_LIST);
```

Comment:
not sure if you want the expected behavior to be to bring the user to the list view if the user adds the application to the applicant on its profile panel.